print('--------------Tema12: TRY Except(Klaidų tvarkymas, išimtys) 24/04/04-------------------------')

print('---------------------------------Uzdaviniai: functions repeating--------------------------------')
"""
Pakartokite Funkcijos ir Funkcijos (2 dalis), kad užbaigtumėte šias užduotis.
"""

# ---------------------------------------------------------------------------------------
"""
1. Sukurkite bent 5 skirtingas funkcijas ir pabandykite apdoroti bent 5 integruotas "Python" išimtis.
"""
print('Uzduotis 1')
# ---------------------------------------------------------------------------------------
"""
2.Python kalboje, dalijant iš nulio, gauname ZeroDivisionError. Jūsų užduotis - sukurti funkciją, kuri:
    -Kaip argumentus priima du skaičius.
    -Mėgina padalyti pirmąjį skaičių iš antrojo.
    -Jei antrasis skaičius yra nulis, ji turėtų sugauti ZeroDivisionError ir grąžinti pasirinktinį klaidos pranešimą.
    -Jei dalijimas sėkmingas, turėtų būti grąžinamas rezultatas.
    -Nepriklausomai nuo to, ar dalijimas pavyko, ar ne, turėtų būti išspausdintas pranešimas "Attempted division". 
    Jei įvesties duomenys nėra skaičiai, pateikiamas TypeError pranešimas. Funkcija pagauna šią TypeError ir grąžina 
    pasirinktinį klaidos pranešimą.
"""

print('Uzduotis 2')
# ---------------------------------------------------------------------------------------
"""
3.Sukurkite mini "Python" programą, kuri įvestų du skaičius ir grąžintų jų sumą, atimtį, dalybą, daugybą. 
Tvarkykite visas galimas klaidas.
"""

print('Uzduotis 3')
# ---------------------------------------------------------------------------------------
"""
4.Atnaujinkite ankstesnę užduotį su galimomis raise išimtimis.
"""

print('Uzduotis 4')
